lib/Command/Import.php

```diff
@@ -72,33 +68,32 @@ protected function configure(): void {
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$io = new SymfonyStyle($input, $output);
+
 		try {
 			$uid = $input->getOption('user');
 			if (!empty($uid)) {
 				$user = $this->userManager->get($uid);
 				if ($user === null) {
-					$output->writeln("<error>User <$uid> does not exist</error>");
+					$io->error("User <$uid> does not exist");
 					return 1;
 				} else {
-					$question = new ConfirmationQuestion(
-						'Warning: A user with this uid already exists!'."\n"
-						. 'Do you really want to overwrite this user with the imported data? (y/n) ', false);
-					if (!$this->questionHelper->ask($input, $output, $question)) {
-						$output->writeln('aborted.');
+					$io->warning('A user with this uid already exists!');
+					if (!$io->confirm('Do you really want to overwrite this user with the imported data?', false)) {
+						$io->writeln('aborted.');
 						return 1;
 					}
 				}
 			} else {
 				$user = null;
 			}
 			$path = $input->getArgument('archive');
-			$output->writeln("Importing from ${path}…");
+			$io->writeln("Importing from ${path}…");
 			$importSource = new ImportSource($path);
-			$this->migrationService->import($importSource, $user, $output);
-			$output->writeln("Successfully imported from ${path}");
+			$this->migrationService->import($importSource, $user, $io);
+			$io->writeln("Successfully imported from ${path}");
 		} catch (\Exception $e) {
-			$output->writeln("$e");
-			$output->writeln("<error>" . $e->getMessage() . "</error>");
+			$io->error($e->getMessage());
```

Comment:
Did you remove the trace on purpose?

Reply:
Yes, seemed too verbose but if you think we should add the trace output back here and in the export command then we could add it back?

Reply:
Maybe add it in high verbose level then?